### ISSUE TITLE
paintable powerline ain't paintable on dedicated server

### ISSUE THREAD

**Issue by dadoirie** [DavidHGillen/Satisfactory_AB-CableChoices#12]
was just quickly testing the mod on my dedicated server I'm playing with a friend and discovered that the paintable powerline is targetable and clickable by the customizer and does color it, but when not hovering it anymore the color is the standard one when building. basically powerline ain't colorable on dedicated server (linux - can't say for windows)

**Comment by DavidHGillen** [comment]
Will investigate

**Comment by rschre** [comment]
> basically powerline ain't colorable on dedicated server (linux - can't say for windows)

Just tested on windows dedicated server: it behaves exactly the same

**Comment by dadoirie** [comment]
then its only on linux dedicated servers where its failing for whatever reason ... might be difficult to determine the cause of it - feel free closing this issue since its not that big of a dealbreaker - there are more serious issues which require attention (for example fluidextras ai packager on dedicated server, but I'll make a issue in the corresponding repo)

**Comment by rschre** [comment]
Sorry for being unclear, I meant to say that it behaves the way you described and coloring isn't working properly in Windows server as well.

**Comment by DavidHGillen** [comment]
I'd love an update for how this is doing with 1.0.7 as I've had to hook some vanilla functions to improve behaviours and I don't have dedicated servers to test with.

**Comment by dadoirie** [comment]
sadly no - still not keeping the color - it is targetable, but when clicking on it just doesn't keep the color

**Comment by Aaronstar-McClure** [comment]
Hey, if you are looking for a dedicated server to test mod development on, let me know, and I'd be glad to provide one.

**Comment by Phishinabowl** [comment]
Hey David just wondering if you were able to make any progress with this on dedicated servers. The mod WORKS, its just the customizable power cables that seem to not want to take the customization on dedicated servers.

**Comment by DavidHGillen** [comment]
Life is being life. Still have every intention of solving this for everyone but no specific ETA

**Comment by dayjobtitus** [comment]
Confirming this is an issue on dedicated servers including windows and Linux. I am still loving the mod for all the other purposes, but painting wires will not work, nor applying the cool skin for wires. It of course does work fine on a local game instance but even if I save the game with the wires colored, and upload it to the server and join it, the colors disappear from the cables.

**Comment by DavidHGillen** [comment]
Yup. I'm finally done with the fire that has kept me busy for the last while. Will be taking the Ficsmas break to see if I can finally pin this down

**Comment by The-Toblin** [comment]
Can confirm that the dedicated server issue with customization not sticking remains. However, the mod seems to be working perfectly fine in 1.1 aside from that issue.

**Comment by w1dg3r** [comment]
> Can confirm that the dedicated server issue with customization not sticking remains. However, the mod seems to be working perfectly fine in 1.1 aside from that issue.

Same here, tested both dedicated and local. Local has no issues but the dedicated is just the default powercable color
